apps/coscrad-frontend/src/components/resource-info/presenters/resource-info.presenter.tsx

```diff
@@ -36,8 +37,13 @@ IAggregateInfo<ResourceType> & { route: string }): JSX.Element => {
                         {/* For the `xs` see https://github.com/mui/material-ui/issues/11339
                         Seems like it's still broken in @material-ui/core ^4.12.3 */}
                         <Grid item zeroMinWidth xs>
-                            <Typography variant="h6" color="primary" fontWeight="bold">
-                                {label}
+                            <Typography
+                                variant="h6"
+                                color="primary"
+                                textTransform="capitalize"
```

Comment:
Just curious, don't we already have caps by this point?

Reply:
Good catch. As a general principle let's not use text transforms like this. Let's remove this.

Reply:
done